### ISSUE TITLE
iSpindel not logging data in Fermentrack for several days

### ISSUE THREAD

**Issue by suchidog** [thorrak/fermentrack#435]
Hi,

I am having an issue where my iSpindel's start logging data to Fermentrack and then drop off for up to four days before logging data again. Upon inspection when dry hopping the status light on the iSpindel is flashing. Could this be an  issue where Fermentrack is not 'awake' for that small window of time when the iSpindel transits data?
![87398691_10163012685695608_7811158116728307712_o](https://user-images.githubusercontent.com/58539258/75127729-6392a380-570c-11ea-859c-7bb54c8a2eb8.jpg)

Thanks

**Comment by mp-se** [comment]
I dont think this is an issue on fermentrack. It looks more like a network problem. Are you fermenting in a steel fermenter? Have you checked the network coverage from where you ferment?

**Comment by thorrak** [comment]
@mp-se submitted a PR which adds a new field to the iSpindel page showing the timestamp data was last received from an iSpindel. This PR will be merged into `dev` after #462 (the Django 3.0 upgrade) goes live into `master` (so ~Friday, June 5th)

**Comment by thorrak** [comment]
This is now merged into dev - I'll likely do another push to master in about ~1 month which will contain this as well
